mods/cnc/maps/gdi09/gdi09-AI.lua

```diff
@@ -0,0 +1,242 @@
+--[[
+   Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+AttackPaths = { { waypoint7 }, { waypoint8 } }
```

Comment:
Can we get an empty line between the end of the header and the first code line, please?